highfidelity/hifi
# Fix orphaned particles left after making a user connection.

Fix particles that are occasionally left over after user makes a connection.

To test:

Needs more than one avatar.

Both or all should run this pr.

Make multiple connection attempts using either or both controllers/extend hand with 'x' key.

Mess with the hand positions to break the connection early. Grip and quickly release etc.

Test with existing connections and avatars that are not connected.

Have an observer check that no orphan particles are left after making a connection or confirming a connection. 

I think this issue should be on Manuscript but I don't have full access.
